JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -479,34 +480,65 @@ private fun InterestsTabRow(
     updateSection: (Sections) -> Unit,
     tabContent: List<TabContent>
 ) {
-    // TODO: Make the tabs narrower and aligned to the left in large screens
-    TabRow(
-        selectedTabIndex = selectedTabIndex,
-        backgroundColor = MaterialTheme.colors.onPrimary,
-        contentColor = MaterialTheme.colors.primary
-    ) {
-        tabContent.forEachIndexed { index, tabContent ->
-            val colorText = if (selectedTabIndex == index) {
-                MaterialTheme.colors.primary
-            } else {
-                MaterialTheme.colors.onSurface.copy(alpha = 0.8f)
+    BoxWithConstraints {
+        val windowSize = remember(maxWidth) { getWindowSize(maxWidth) }
+        when (windowSize) {
+            WindowSize.Compact -> {
+                TabRow(
+                    selectedTabIndex = selectedTabIndex,
+                    backgroundColor = MaterialTheme.colors.onPrimary,
+                    contentColor = MaterialTheme.colors.primary
+                ) {
+                    InterestsTabRowContent(selectedTabIndex, updateSection, tabContent)
+                }
             }
-            Tab(
-                selected = selectedTabIndex == index,
-                onClick = { updateSection(tabContent.section) },
-                modifier = Modifier.heightIn(min = 48.dp)
-            ) {
-                Text(
-                    text = stringResource(id = tabContent.section.titleResId),
-                    color = colorText,
-                    style = MaterialTheme.typography.subtitle2,
-                    modifier = Modifier.paddingFromBaseline(top = 20.dp)
-                )
+            else -> {
+                ScrollableTabRow(
+                    selectedTabIndex = selectedTabIndex,
+                    backgroundColor = MaterialTheme.colors.onPrimary,
+                    contentColor = MaterialTheme.colors.primary,
+                    edgePadding = 0.dp
+                ) {
+                    InterestsTabRowContent(
+                        selectedTabIndex = selectedTabIndex,
+                        updateSection = updateSection,
+                        tabContent = tabContent,
+                        modifier = Modifier.padding(horizontal = 8.dp)
+                    )
+                }
             }
         }
     }
 }
 
+@Composable
+private fun InterestsTabRowContent(
+    selectedTabIndex: Int,
+    updateSection: (Sections) -> Unit,
+    tabContent: List<TabContent>,
+    modifier: Modifier = Modifier
+) {
+    tabContent.forEachIndexed { index, content ->
+        val colorText = if (selectedTabIndex == index) {
+            MaterialTheme.colors.primary
+        } else {
+            MaterialTheme.colors.onSurface.copy(alpha = 0.8f)
```

Comment:
Should this be an emphasis instead? What's the material guide for tabs?

Reply:
As discussed, this code just moved, this alpha wasn't specifically added. We can address this in a different PR.

Reply:
As discussed offline, let's leave it as is for now. Can revisit in the future, thank you!